src/shader_recompiler/backend/glsl/glsl_emit_context.cpp

```diff
@@ -458,9 +458,10 @@ void EmitContext::DefineGenericOutput(size_t index, u32 invocations) {
         std::string definition{fmt::format("layout(location={}", index)};
         const u32 remainder{4 - element};
         const TransformFeedbackVarying* xfb_varying{};
-        if (!runtime_info.xfb_varyings.empty()) {
-            xfb_varying = &runtime_info.xfb_varyings[base_index + element];
-            xfb_varying = xfb_varying && xfb_varying->components > 0 ? xfb_varying : nullptr;
+        size_t xfb_varying_index{base_index + element};
```

Comment:
```suggestion
        const size_t xfb_varying_index{base_index + element};
```

For consistency with the vars above it. Ditto for the other file as well